pkg/operator/podsecurityreadinesscontroller/podsecurityreadinesscontroller.go

```diff
@@ -114,58 +100,23 @@ func (c *PodSecurityReadinessController) sync(ctx context.Context, syncCtx facto
 	return err
 }
 
-func (c *PodSecurityReadinessController) isNamespaceViolating(ctx context.Context, ns *corev1.Namespace) (bool, error) {
-	if ns.Labels[psapi.EnforceLevelLabel] != "" {
-		// If someone has taken care of the enforce label, we don't need to
-		// check for violations. Global Config nor PS-Label-Syncer will modify
-		// it.
-		return false, nil
-	}
-
-	targetLevel := ""
-	for _, label := range podSecurityAlertLabels {
-		levelStr, ok := ns.Labels[label]
-		if !ok {
-			continue
-		}
-
-		level, err := psapi.ParseLevel(levelStr)
-		if err != nil {
-			klog.V(4).InfoS("invalid level", "namespace", ns.Name, "level", levelStr)
-			continue
-		}
-
-		if targetLevel == "" {
-			targetLevel = levelStr
-			continue
-		}
-
-		if psapi.CompareLevels(psapi.Level(targetLevel), level) < 0 {
-			targetLevel = levelStr
-		}
-	}
-
-	if targetLevel == "" {
-		// Global Config will set it to "restricted".
-		targetLevel = string(psapi.LevelRestricted)
-	}
-
-	nsApply := applyconfiguration.Namespace(ns.Name).WithLabels(map[string]string{
-		psapi.EnforceLevelLabel: string(targetLevel),
-	})
-
-	_, err := c.kubeClient.CoreV1().
-		Namespaces().
-		Apply(ctx, nsApply, metav1.ApplyOptions{
-			DryRun:       []string{metav1.DryRunAll},
-			FieldManager: "pod-security-readiness-controller",
-		})
+func getNamespaceSelector() (string, error) {
```

Comment:
- func signature hides what the ns selector will select
- `get` prefixes do not feel very go-like

How about something like `psaEnforcedNamespaceSelector()`? or passing arguments, like `namespaceLabelSelector(psapi.EnforceLevelLabel, selection.DoesNotExist)`?

Reply:
It is a pure man's constant 😄